<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnPorts">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ports</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbPorts">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenPorts">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnMessage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lstMessages">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>90</y>
      <width>201</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
